en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -131,6 +132,22 @@ To repair a connection:
 . Click the change icon in the trigger or action that requires repair, and then click *Change Connection*.
 . Click *Repair*.
 
+== Delete a Connection
+
+If you no longer need a specific OAuth connection, you can delete it.
+
+When you attempt to delete a connection, Composer ensures that deleting the connection will not affect other flows.
```

Comment:
I would propose a reword here. Technically it can't ensure that it will not affect other flows, it can just warn the user as to which flows will be affected.

Reply:
In fact, it will affect flows, flows which are using this connection will now be stopped, instead of just running and failing due to invalid connection.